Community/WeatherStationUploader/automation/jsr223/community/weatherStationUploader.py

```diff
@@ -25,16 +25,14 @@
 __version_info__ = tuple([ int(num) for num in __version__.split('.')])
 
 import os, time, math
+from meteocalc import Temp, dew_point, heat_index
+import org.joda.time.DateTime as DateTime
+import org.joda.time.DateTimeZone as DateTimeZone
```

Comment:
Replace this line and the one above it with...
```
from org.joda.time import DateTime, DateTimeZone
```

Reply:
Fixed